### PR TITLE
bin scripts to populate oneclick catalog and run deltas

### PR DESCRIPTION
**Pull request by daryanypl**

### CODE REVIEW

File: bin/oneclick_library_delta [hunk 1]
```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python
+"""Update the circulation manager server with new books from the
+open access content server."""
```
Comment (leonardr) [inline bin/oneclick_library_delta]: Docstring needs updating.

File: bin/oneclick_library_delta [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+package_dir = os.path.join(bin_dir, "..")
+sys.path.append(os.path.abspath(package_dir))
+from core.scripts import OneClickDeltaScript
+
+from api.config import Configuration
+Configuration.load()
+library_id = Configuration.integration(Configuration.ONECLICK_INTEGRATION).get("library_id", None)
+
+print "Applying catalog delta of library #%s" % library_id
+OneClickDeltaScript().run()
```
Comment (leonardr) [inline bin/oneclick_library_delta]: If you need to print this string, put it inside `OneClickDeltaScript` and have it use `self.log`.

File: bin/oneclick_library_import [hunk 3]
```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env python
+"""Update the circulation manager server with new books from the
+open access content server."""
```
Comment (leonardr) [inline bin/oneclick_library_import]: Again, docstring needs changing.

**Comment by leonardr** [comment]
Looks good with minor changes. Incorporate the fix from https://github.com/NYPL-Simplified/circulation/pull/424/commits/48f0913df4d14e0c40dc66b66023957e829c3ee6 to get the tests to pass.

File: bin/oneclick_library_delta (hunk 1 above)
Comment (daryanypl) [inline bin/oneclick_library_delta]: done

File: bin/oneclick_library_delta (hunk 2 above)
Comment (daryanypl) [inline bin/oneclick_library_delta]: I've seen this print statement in another script, and I liked it.  It lets the caller know that something is happening, before we try to get into the actual Script class, and right on the command line.  I agree that most stuff should be put neatly away in logs, but for this one statement, I kinda like it nice and visible to stdout?

File: bin/oneclick_library_import (hunk 3 above)
Comment (daryanypl) [inline bin/oneclick_library_import]: done

File: bin/oneclick_library_delta (hunk 2 above)
Comment (leonardr) [inline bin/oneclick_library_delta]: If you were just printing something out it wouldn't be a big deal, but you're loading the site configuration and looking something up first. That's what I object to.

But I'm not going to hold up the branch on this issue.

**Comment by leonardr** [comment]
:+1:

**Pull request merged**